Sources/LSPLogging/CustomLogStringConvertible.swift

```diff
@@ -67,3 +67,21 @@ extension String {
     return Insecure.MD5.hash(data: Data(self.utf8)).description
   }
 }
+
+private struct OptionalWrapper<Wrapped>: CustomLogStringConvertible where Wrapped: CustomLogStringConvertible {
```

Comment:
Were optionals just redacted previously 😅?

Reply:
We never tried to log an optional `CustomLogStringConvertible` before, so the question just didn’t come up.